config/site_settings.yml

```diff
@@ -996,6 +996,7 @@ posting:
     default: false
   show_published_pages_login_required:
     default: false
+  skip_auto_delete_reply_likes: 5
```

Comment:
setting needs localization for the description :) 

"When automatically deleting old replies, skip deleting posts with this number of likes or more."